docs/shortcuts.html

```diff
@@ -1,664 +1,837 @@
-              <td> <span class="kbd">Opt</span> + <span class="kbd">W</span> </td>
-            </tr>
-            <tr>
-              <td> Run snippet </td>
-              <td> <span class="kbd">Ctrl</span> + <span class="kbd">Enter</span> </td>
-              <td> <span class="kbd">Cmd</span> + <span class="kbd">Enter</span> </td>
-            </tr>
-            <tr>
-              <td> Toggle comment </td>
-              <td> <span class="kbd">Ctrl</span> + <span class="kbd">/</span> </td>
-              <td> <span class="kbd">Cmd</span> + <span class="kbd">/</span> </td>
-            </tr>
-          </tbody>
-        </table>
-        <p>
-          <img src="../images/pause-gray.png" alt="Pause on Exception Button" />
-          Don't pause on exceptions
-        </p>
-        <p>
-          <img src="../images/pause-blue.png" alt="Pause on All Exceptions" />
-          Pause on All exceptions (including those caught within try/catch blocks)
-        </p>
-        <p>
-          <img src="../images/pause-purple.png" alt="Pause on Uncaught Exceptions" />
-          Pause on uncaught exceptions (usually the one you want)
-        </p>
+  <h2 id="code-editor"> Code Editor Shortcuts </h2>
+
+  <table>
+    <tbody>
+      <tr>
+        <th></th>
+        <th> Common </th>
+      </tr>
+      <tr>
+        <td> Go character left </td>
+        <td> <span class="kbd">Left arrow</span> </td>
+      </tr>
+      <tr>
+        <td> Go character right </td>
+        <td> <span class="kbd">Right arrow</span> </td>
+      </tr>
+      <tr>
+        <td> Go line up </td>
+        <td> <span class="kbd">Up arrow</span> </td>
+      </tr>
+      <tr>
+        <td> Go line down </td>
+        <td> <span class="kbd">Down arrow</span> </td>
+      </tr>
+      <tr>
+        <td> Go to end of line </td>
+        <td> <span class="kbd">End</span> </td>
+      </tr>
+      <tr>
+        <td> Go line start smart </td>
+        <td> <span class="kbd">Home</span> </td>
+      </tr>
+      <tr>
+        <td> Page up </td>
+        <td> <span class="kbd">PageUp</span> </td>
+      </tr>
+      <tr>
+        <td> Page down </td>
+        <td> <span class="kbd">PageDown</span> </td>
+      </tr>
+      <tr>
+        <td> Delete character after </td>
+        <td> <span class="kbd">Delete</span> </td>
+      </tr>
+      <tr>
+        <td> Delete character before </td>
+        <td> <span class="kbd">Backspace</span> </td>
+      </tr>
+      <tr>
+        <td> Default tab </td>
+        <td> <span class="kbd">Tab</span> </td>
+      </tr>
+      <tr>
+        <td> Indent less </td>
+        <td> <span class="kbd">Shift</span> + <span class="kbd">Tab</span> </td>
+      </tr>
+      <tr>
+        <td> Smart new line and indent </td>
+        <td> <span class="kbd">Enter</span> </td>
+      </tr>
+      <tr>
+        <td> Autocomplete </td>
+        <td> <span class="kbd">Ctrl</span> + <span class="kbd">Space</span> </td>
```

Comment:
For me, this isn't even necessary i think. autocomplete just auto pops up when i start typing a keyword